check-for-diff/action.yml

```diff
@@ -0,0 +1,14 @@
+name: Check for diff
```

Comment:
Maybe `Check git diff` would be a bit more precise?

Reply:
Agreed! f0f5c6ba72d552a35e448b474dcca2558477b3b4